README.md

```diff
@@ -1,2 +1,92 @@
 # django-nepali
+
 A django package on top of 'nepali' a python package which supports nepali date time, time conversion, etc on django projects.
```

Comment:
Let's add a hyperlink on the `nepali` package for better reachability

Reply:
.

Reply:
Updated